contracts/modules/staking/StakingProducts.sol

```diff
@@ -518,15 +517,15 @@ contract StakingProducts is IStakingProducts, MasterAwareV2, Multicall {
 
   function _setInitialProducts(uint poolId, ProductInitializationParams[] memory params) internal {
 
-    uint globalMinPriceRatio = cover().getGlobalMinPriceRatio();
+    uint defaultMinPriceRatio = cover().getDefaultMinPriceRatio();
     uint totalTargetWeight;
 
     for (uint i = 0; i < params.length; i++) {
 
       ProductInitializationParams memory param = params[i];
 
-      if (params[i].targetPrice < globalMinPriceRatio) {
-        revert TargetPriceBelowGlobalMinPriceRatio();
+      if (params[i].targetPrice < defaultMinPriceRatio) {
```

Comment:
This should also check for the product price which may be lower than the default one (`coverProducts.getMinPrices()`). This is correctly implemented in the `setProducts` function.